QUEST_LV_0100.tsv

```diff
@@ -6300,27 +6300,27 @@ QUEST_LV_0100_20150317_006295	{memo X}
 QUEST_LV_0100_20150317_006296	{memo X}
 QUEST_LV_0100_20150317_006297	Defeat the monsters that reacted to the ominous energy
 QUEST_LV_0100_20150317_006298	The surrounding monsters turned aggressive when an ominous energy fumed from the Ershike Altar. Defeat those monsters.
-QUEST_LV_0100_20150317_006299	Check the obelisk
-QUEST_LV_0100_20150317_006300	Check the obelisk in Slepingas Stream.. 
+QUEST_LV_0100_20150317_006299	Check the Obelisk
+QUEST_LV_0100_20150317_006300	Check the Obelisk in Slepingas Stream.. 
 QUEST_LV_0100_20150317_006301	Defeat the Blue Woodspirit
 QUEST_LV_0100_20150317_006302	$Find the Priest of Andale Village
 QUEST_LV_0100_20150317_006303	{memo X}
 QUEST_LV_0100_20150317_006304	Collect the Languid Herbs.
-QUEST_LV_0100_20150317_006305	Let's create Languid Bomb to defeat monsters threatening Andale Village. Collect Languid Flowers need for making the bomb.
-QUEST_LV_0100_20150317_006306	$Talk to Andale Village Disciple
-QUEST_LV_0100_20150317_006307	Gathered enough Languid Flowers. Talk to the Village Disciple in Andale Village. 
-QUEST_LV_0100_20150317_006308	Ask the disciple what else is needed to make the Languid Bomb.
-QUEST_LV_0100_20150317_006309	Collect the strongly scented soul flower.
+QUEST_LV_0100_20150317_006305	Let's create a Languid Bomb to defeat monsters threatening Andale Village. Collect Languid Herbs needed to make the bomb.
+QUEST_LV_0100_20150317_006306	$Talk to the Andale Village Disciple
+QUEST_LV_0100_20150317_006307	$Gathered enough Languid Herbs. Talk to the Village Disciple in Andale Village. 
+QUEST_LV_0100_20150317_006308	Ask the Disciple what else is needed to make the Languid Bomb.
```

Comment:
Necessary to capitalize this (along with Believer)? Because isn't 'disciple' a common noun? It would be different if it said 'Disciple X'.